### ISSUE TITLE
motioneye doesn't work with video0

### ISSUE THREAD

**Issue by thelostinthought** [SensorsIot/IOTstack#85]
.templates/motioneye/service.yml has an issue where the last 2 lines are commented out, causing usb cameras to not work.  uncommenting, or adding an option in ./menu.sh to toggle comments, or warning the user that this happened for security would be nice.

**Comment by Slyke** [comment]
Yep, this will be fixed in the menu rewrite. I'm actually at MotionEye right now. It will leave those lines commented out if it cannot detect `/dev/video0`. It will put those lines in if it can.
